### PR TITLE
added dikshagoyal26 to finalist

### PR DESCRIPTION
**Pull request by dikshagoyal26**
Hi Mentor,

I have completed All 5 projects and 3 Blogs, please review and accept my pull request.
Thank you.

### CODE REVIEW

File: content/finalist/ttjc@finalist@dikshagoyal26/index.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+### Github Finder
+
+Github finder is a webapp to find github users on the basis of their github handle. It displays user's public github information and top 10 repositories of the user. Concept of debouncing is used while fetching books
+
+- **Stack** `Vanilla Js`, `Bootstrap`
+
+- [**hosted link**](https://reallo.netlify.com/)
+
+- [**github link**](https://github.com/Megha-Pathak/Reallo)
```
Comment (gkanishk) [inline content/finalist/ttjc@finalist@dikshagoyal26/index.md:76]: So hilarious how can you copy projects!!!!!!!!

Comment (dikshagoyal26) [reply content/finalist/ttjc@finalist@dikshagoyal26/index.md:76]: shit sorry, the project isn't copied the I just copied the template. Let me make the change and push again.

Comment (gkanishk) [reply content/finalist/ttjc@finalist@dikshagoyal26/index.md:76]: that won't work, better luck next time!!

Comment (dikshagoyal26) [reply content/finalist/ttjc@finalist@dikshagoyal26/index.md:76]: https://github.com/tanaypratap/teamtanay.jobchallenge.dev/pull/1109
I am really very sorry about this. I have created a pull request please do review it.

**Pull request closed without merging**